modules/nw-networkpolicy-deny-all-allowed.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-multi-network-policy.adoc
+:name: network
+:role: admin
+ifeval::[{product-version} >= 4.6]
+:ovn:
+endif::[]
+ifeval::["{context}" == "configuring-multi-network-policy"]
+:multi:
+:name: multi-network
+:role: cluster-admin
+endif::[]
+
+:_content-type: PROCEDURE
+[id="nw-networkpolicy-deny-all-multi-network-policy_{context}"]
+= Creating a default deny all {name} policy
+
+This is a fundamental policy, blocking all cross-pod networking other than network traffic allowed by the configuration of other deployed network policies. This procedure enforces a default `deny-by-default` policy.
+
+[NOTE]
+====
+If you log in with a user with the `cluster-admin` role, then you can create a network policy in any namespace in the cluster.
+====
+
+.Prerequisites
+
+* Your cluster uses a cluster network provider that supports `NetworkPolicy` objects, such as
+ifndef::ovn[]
+the OpenShift SDN network provider with `mode: NetworkPolicy` set.
+endif::ovn[]
+ifdef::ovn[]
+the OVN-Kubernetes network provider or the OpenShift SDN network provider with `mode: NetworkPolicy` set.
+endif::ovn[]
+This mode is the default for OpenShift SDN.
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `{role}` privileges.
+* You are working in the namespace that the {name} policy applies to.
+
+.Procedure
+
+. Create the following YAML that defines a `deny-by-default` policy to deny ingress from all pods in all namespaces. Save the YAML in the `deny-by-default.yaml` file:
++
+[source,yaml]
+----
+ifndef::multi[]
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+endif::multi[]
+ifdef::multi[]
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+endif::multi[]
+metadata:
+  name: deny-by-default
+  namespace: default <1>
+ifdef::multi[]
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: <network_name>
+endif::multi[]
+spec:
+  podSelector: {} <2>
+  ingress: [] <3>
+----
+<1> `namespace: default` deploys this policy to the `default` namespace.
+<2> `podSelector:` is empty, this means it matches all the pods. Therefore, the policy applies to all pods in the default namespace.
+<3> There are no `ingress` rules specified. This causes incoming traffic to be dropped to all pods.
++
+ifdef::multi[]
+--
+where
+
+`<network_name>` specifies the name of a network attachment definition.
```

Comment:
This is the one that's not rendering right. You either need all of the annotations to be in the definition list or all of them to use callouts.

Reply:
You could move them all into the definition list.

Reply:
I wrapped in more expansive ifdef statements